index.bs

```diff
@@ -4215,8 +4215,9 @@ Attributes</h4>
 		Note that increasing {{AnalyserNode/fftSize}} does mean that the
 		<a>current time-domain data</a> must be expanded to include past
 		frames that it previously did not. This means that the
-		{{AnalyserNode}} effectively MUST keep around the last 32768 sample-
-		frames and the <a>current time-domain data</a> is the most recent
+		{{AnalyserNode}} effectively MUST keep around the last
+		32768 sample-frames and the <a>current time-domain
+		data</a> is the most recent
```

Comment:
Just line changes?

Reply:
Oops.  Previously, "sample-" was the end of the line and "frames" was on the next line.  This gets rendered as "sample- frame" (note the space).  I just arranged the line so that "sample-frames" occurs together so that bikeshed can break it at the right place if needed.